### PR TITLE
docs: fix some underdescription or error description

### PR DESCRIPTION
**Pull request by songzxc789**

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-paas?pullRequest=1545) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/blueking-paas?pullRequest=1545) before we can accept your contribution.<br/><hr/>**v_sxdgsong** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/blueking-paas?pullRequest=1545) it.</sub>

File: apiserver/paasng/support-files/apigw/api_doc/zh/upload_source_package_via_link.md [hunk 1]
```diff
@@ -20,26 +20,35 @@
 
 ### 请求示例
 ```bash
-curl -X POST 'http://bkapi.example.com/api/bkpaas3/stag/bkapps/applications/sundy820/modules/default/source_package/link/' \
-  -H 'X-BKAPI-AUTHORIZATION: {"access_token": "your access_token"}' \
-  -H 'Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryohLfLFHX1EPDP7rB' \
-  --data-binary $'------WebKitFormBoundaryohLfLFHX1EPDP7rB\r\nContent-Disposition: form-data; name="package_url"\r\n\r\n   你的源码包链接  \r\n------WebKitFormBoundaryohLfLFHX1EPDP7rB\r\nContent-Disposition: form-data; name="version"\r\n\r\n  你的版本号  \r\n------WebKitFormBoundaryohLfLFHX1EPDP7rB\r\nContent-Disposition: form-data; name="allow_overwrite"\r\n\r\nfalse\r\n------WebKitFormBoundaryohLfLFHX1EPDP7rB--\r\n'
+curl -X POST -H 'Content-Type: application/json' -H 'X-Bkapi-Authorization: {"bk_app_code": "apigw-api-test", "bk_app_secret": "***", "bk_ticket": "***"}' -d '{ "package_url": "https://example.com/generic/example.tar.gz", "version": "0.0.5" }' --insecure https://paas.example.apigw.o.woav3..com/stag/bkapps/applications/app_code/modules/default/source_package/link/
```
Comment (jiayuan929) [inline apiserver/paasng/support-files/apigw/api_doc/zh/upload_source_package_via_link.md:23]: 请求示例里面的 url 统一用：`http://bkapi.example.com/api/bkpaas3`

这里也就是：
`http://bkapi.example.com/api/bkpaas3/stag/bkapps/applications/app_code/modules/default/source_package/link/`

File: apiserver/paasng/support-files/apigw/api_doc/en/search_standard_log_with_get.md [hunk 2]
```diff
@@ -28,44 +29,64 @@ curl -X GET -H 'X-Bkapi-Authorization: {"bk_app_code": "apigw-api-test", "bk_app
...
 }
 ```
 
-### Response Result Parameter Description
+#### Exception Response
+example 1
+```
+{
+    "code": "QUERY_LOG_FAILED",
+    "detail": "查询日志失败: 日志平台接口返回异常<采集项space_84_bklog_ceshichajian__default__json采集链路data_name重复（3631113）>, 请稍后重试"
```
Comment (jiayuan929) [inline apiserver/paasng/support-files/apigw/api_doc/en/search_standard_log_with_get.md:58]: 这样写就好, 不用把完整的信息写出来

```
"detail": "查询日志失败: ..."
```

File: apiserver/paasng/support-files/apigw/api_doc/en/search_standard_log_with_get.md [hunk 3]
```diff
@@ -28,44 +29,64 @@ curl -X GET -H 'X-Bkapi-Authorization: {"bk_app_code": "apigw-api-test", "bk_app
...
+}
+```
+example 2
+```
+{
+    "code": "VALIDATION_ERROR",
+    "detail": "time_range: 该字段是必填项。",
```
Comment (jiayuan929) [inline apiserver/paasng/support-files/apigw/api_doc/en/search_standard_log_with_get.md:65]: `该字段是必填项。`: 这个在英文下应该也是返回英文信息的吧

**Review by jiayuan929 — APPROVED** [review]

**Pull request merged**